README.md

```diff
@@ -40,3 +42,11 @@ If you have shipped open source software and/or UX-heavy web apps before, please
 ![](./mocks/images/depviz.mocks.011.jpg)
 ---
 ![](./mocks/images/depviz.mocks.012.jpg)
+
+## Contribute
+
+PRs are accepted! Please dive in to [the issues](https://github.com/jbenet/depviz/issues), too.
```

Comment:
We probably [want][1] a [`CONTRIBUTING.md`][2] too.  And maybe we should pull in more of the Protocol/IPFS boilerplate ([DCO][3], [CoC][4], [JS guidelines][5], …?).  I'm fine not handling that in this PR, but thought I'd mention it in case we wanted to start now ;).

[1]: https://help.github.com/articles/setting-guidelines-for-repository-contributors/
[2]: https://github.com/blog/1184-contributing-guidelines
[3]: https://github.com/ipfs/community/blob/master/docs/developer-certificate-of-origin
[4]: https://github.com/ipfs/community/blob/master/code-of-conduct.md
[5]: https://github.com/ipfs/community/blob/master/js-project-guidelines.md